### ISSUE TITLE
Update tutorial "One-class segmentation with 2D U-Net"

### ISSUE THREAD

**Issue by mariehbourget** [ivadomed/ivadomed#717]
Following the integration of the new loader (#699), the tutorial [One-class segmentation with 2D U-Net](https://ivadomed.org/en/latest/tutorials/one_class_segmentation_2d_unet.html#one-class-segmentation-with-2d-u-net) needs to be updated to match the new split (random seed is now applied to filenames and not participant_id so the split is different).

TODO:

- [ ] Run the tutorial
- [ ] Update the terminal output example in [Train model](https://ivadomed.org/en/latest/tutorials/one_class_segmentation_2d_unet.html#train-model)
- [ ] Update the terminal output example in [Evaluate model](https://ivadomed.org/en/latest/tutorials/one_class_segmentation_2d_unet.html#evaluate-model)
- [ ] Update the filenames in the fsleyes command in Evaluate model
- [ ] Update the output and ground truth segmentations images in Evaluate model

**Comment by konantian** [comment]
@mariehbourget From the original tutorial, I noticed that there are 10 subjects under the data_example_spinegeneric. However, for the evaluation step, only two subjects evaluated, do you have any ideas why is this and how to evaluate the whole 10 subjects?

**Comment by charleygros** [comment]
> only two subjects evaluated, do you have any ideas why is this and how to evaluate the whole 10 subjects?

It is probably because the evaluation step is only run on the testing dataset (which is a subset of the training dataset): see [here ](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json#L40)for split: 60% for training, 20% for testing --> which matches up with "2 subjects"
